litex/tools/litex_json2dts_linux.py

```diff
@@ -285,8 +285,8 @@ def generate_dts(d, initrd_start=None, initrd_size=None, polling=False):
     ethmac_csr_base  = d["csr_bases"]["ethmac"],
     ethmac_mem_base  = d["memories"]["ethmac"]["base"],
     ethmac_mem_size  = d["memories"]["ethmac"]["size"],
-    ethmac_tx_slots  = d["constants"]["ethmac_tx_slots"],
-    ethmac_rx_slots  = d["constants"]["ethmac_rx_slots"],
+    ethmac_tx_depth  = d["constants"]["ethmac_tx_slots"] * d["constants"]["ethmac_slot_size"],
```

Comment:
This shouldn't be the slot size, but the size of the fifo. So for the default case where we have two slots, the fifo-depth should be 0x1000.